src/dashboard/components/SignUp/SignUp.tsx

```diff
@@ -24,12 +25,12 @@ export function SignUp() {
 
     // functions will be used to validate values at corresponding key
     validate: {
-      username: (value) =>
-        value.length < 2 ? 'Name must have at least 2 letters' : mutation.isError,
-      password: (value) =>
-        value.length < 6 ? 'Password must have at least 6 characters' : mutation.isError,
+      username: (value) => validateUsername(value) || mutation.isError,
```

Comment:
Is mutation ever going to be stale?

Reply:
ahhhhh, the bug was here. Nice catch, was wondering why the form wasn't submitting